AutomaticInterface/Tests/GeneratorTests.cs

```diff
@@ -1260,16 +1091,13 @@ public string CMethod<T, T1, T2, T3, T4>(string x, string y)
             // </auto-generated>
             //--------------------------------------------------------------------------------------------------
 
-            using System.CodeDom.Compiler;
-            using AutomaticInterface;
-
             namespace AutomaticInterfaceExample
             {
-                [GeneratedCode("AutomaticInterface", "")]
+                [global::System.CodeDom.Compiler.GeneratedCode("AutomaticInterface", "")]
                 public partial interface IDemoClass
                 {
                     /// <inheritdoc />
-                    string CMethod<T, T1, T2, T3, T4>(string x, string y) where T : class where T1 : struct where T3 : DemoClass where T4 : IDemoClass;
+                    string CMethod<T, T1, T2, T3, T4>(string x, string y) where T : class where T1 : struct where T3 : global::AutomaticInterfaceExample.DemoClass where T4 : IDemoClass;
```

Comment:
This is the one place where types aren't fully qualified - `IDemoClass` doesn't exist at the time of generation, so it can't be resolved. This is OK though, since the reference is within the scope of `IDemoClass`, so no collision is possible